--- a/setup.py	2014-04-18 17:38:07.000000000 -0600
+++ b/setup.py	2014-04-27 18:13:12.468820716 -0600
@@ -8,7 +8,7 @@
 setup(
     name='rosdistro',
     version=__version__,
-    install_requires=['argparse', 'catkin_pkg', 'rospkg', 'PyYAML', 'setuptools'],
+    install_requires=['catkin_pkg', 'rospkg', 'PyYAML', 'setuptools'],
     packages=find_packages('src'),
     package_dir={'': 'src'},
     scripts=[
