diff -up ./doc/conf.py.sphinx ./doc/conf.py
--- ./doc/conf.py.sphinx	2014-02-07 03:56:12.000000000 -0500
+++ ./doc/conf.py	2014-02-08 20:35:35.186378222 -0500
@@ -25,8 +25,7 @@ from rosdistro._version import __version
 # They can be extensions coming with Sphinx (named 'sphinx.ext.*') or your
 # custom ones.
 extensions = ['sphinx.ext.ifconfig', 'sphinx.ext.todo', 'sphinx.ext.graphviz',
-              'sphinx.ext.intersphinx', 'sphinx.ext.doctest',
-              'catkin_sphinx.ShLexer', 'catkin_sphinx.cmake']
+              'sphinx.ext.intersphinx', 'sphinx.ext.doctest']
 todo_include_todos = True
 
 # Add any paths that contain templates here, relative to this directory.
@@ -108,13 +107,9 @@ pygments_style = 'sphinx'
 # Add any paths that contain custom themes here, relative to this directory.
 html_theme_path = [os.path.join(os.path.expanduser('~'), 'sphinx'), 'themes']
 
-import catkin_sphinx
-html_theme_path.append(os.path.join(os.path.dirname(catkin_sphinx.__file__),
-                                    'theme'))
 
 # The theme to use for HTML and HTML Help pages.  Major themes that come with
 # Sphinx are currently 'default' and 'sphinxdoc'.
-html_theme = 'ros-theme'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
@@ -130,7 +125,6 @@ html_theme = 'ros-theme'
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-html_logo = 'ros.png'
 
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
@@ -181,7 +175,6 @@ html_logo = 'ros.png'
 #html_file_suffix = ''
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'catkin-cmakedoc'
 
 
 # -- Options for LaTeX output -------------------------------------------------
@@ -194,10 +187,6 @@ htmlhelp_basename = 'catkin-cmakedoc'
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual])
-latex_documents = [
-  ('index', 'catkin.tex', ur'Catkin',
-   ur'troy d. straszheim', 'manual'),
-]
 
 # The name of an image file (relative to this directory) to place at the top of
 # the title page.
