diff -up ./setup.py.setuptools ./setup.py
--- ./setup.py.setuptools	2014-04-08 19:52:15.644204990 -0400
+++ ./setup.py	2014-04-08 19:52:32.050809073 -0400
@@ -8,7 +8,7 @@ exec(open(os.path.join(os.path.dirname(_
 setup(
     name='rosdistro',
     version=__version__,
-    install_requires=['argparse', 'catkin_pkg', 'distribute', 'rospkg', 'PyYAML'],
+    install_requires=['argparse', 'catkin_pkg', 'setuptools', 'rospkg', 'PyYAML'],
     packages=find_packages('src'),
     package_dir={'': 'src'},
     scripts=[
